src/libertem/web/notebook_generator/notebook_generator.py

```diff
@@ -1,5 +1,7 @@
 import io
+
 import nbformat as nbf
+
 from .code_template import CodeTemplate
 
```

Comment:
These notebook_generator files have changed because I ran isort on everything in the web folder. I thought it doesn't hurt to isort this as well.